AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4437,8 +4432,8 @@ function Test-IsCloudKerberosTicketRetrievalEnabled {
 function Debug-EntraKerbAdminConsent {
     [CmdletBinding()]
     param (
-        [Parameter(Mandatory=$True, Position=0, HelpMessage="Storage account name")]
-        [string]$StorageAccountName,
+        [Parameter(Mandatory=$True, Position=0, HelpMessage="Safehost Endpoint for User Account")]
+        [string]$AccountEndpointSafeHost,

```

Comment:
"Safe" refers to DNS: as in, this is a name that you can safely pass to DNS to get an IP. But it's really an implementation detail we don't have to concern ourselves with. I would recommend just calling this `$AccountFileEndpoint`